Sources/CryptoSwift/ChaCha20.swift

```diff
@@ -6,54 +6,25 @@
 //  Copyright (c) 2014 Marcin Krzyzanowski. All rights reserved.
 //
 
-final public class ChaCha20 {
-    
-    public enum Error: ErrorType {
-        case MissingContext
-    }
-    
-    static let blockSize = 64 // 512 / 8
-    private let stateSize = 16
-    private var context:Context?
-    
-    final private class Context {
-        var input:[UInt32] = [UInt32](count: 16, repeatedValue: 0)
-        
-        deinit {
-            for i in 0..<input.count {
-                input[i] = 0x00;
-            }
-        }
-    }
+final public class ChaCha20: Salsa20 {
     
     public init?(key:[UInt8], iv:[UInt8]) {
+        super.init(rounds: 20)
         if let c = contextSetup(iv: iv, key: key) {
             context = c
         } else {
             return nil
         }
     }
     
-    public func encrypt(bytes:[UInt8]) throws -> [UInt8] {
-        guard context != nil else {
-            throw Error.MissingContext
-        }
-        
-        return try encryptBytes(bytes)
-    }
-    
-    public func decrypt(bytes:[UInt8]) throws -> [UInt8] {
-        return try encrypt(bytes)
-    }
-    
-    private final func wordToByte(input:[UInt32] /* 64 */) -> [UInt8]? /* 16 */ {
+    override final func wordToByte(input:[UInt32] /* 64 */) -> [UInt8]? /* 16 */ {
```

Comment:
maybe I'm missing something here, but this seems unecessary to override `wordToByte` - it's indentical to version of `Salsa20`. What I'm missing ?